.devcontainer/devcontainer.json

```diff
@@ -3,11 +3,16 @@
 {
 	"name": "Ubuntu",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+	"build": {
+		"dockerfile": "dockerfile",
+		"options": [
+			"--platform=linux/amd64"
+		]
+	},
```

Comment:
Does this solve the problem running on M1? The original image was already built with linux/amd64

Reply:
https://stackoverflow.com/a/78827072/8323399